hack/bazel-build.sh

```diff
@@ -62,22 +62,24 @@ bazel run \
 
 # build platform native virtctl explicitly
 bazel run \
-    --config=${HOST_ARCHITECTURE} \
     :build-virtctl -- ${CMD_OUT_DIR}/virtctl/virtctl
 
-# cross-compile virtctl for
+# compile virtctl for amd64 and arm64
 
 # linux
 bazel run \
-    --config=${HOST_ARCHITECTURE} \
     :build-virtctl-amd64 -- ${CMD_OUT_DIR}/virtctl/virtctl-${KUBEVIRT_VERSION}-linux-amd64
 
+bazel run \
```

Comment:
Changes look good to me. I just think we may want to change the build flow, that we do not build for all architectures during normal development cycles, since it is at least initially time-consuming.